content/pt-br/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -0,0 +1,353 @@
+  ActiveDirectoryConfig:
+    GroupManagedServiceAccounts:
+    - Name: WebApp1       #Nome de usuário da conta GMSA
+      Scope: CONTOSO      #Nome de Domínio NETBIOS
+    - Name: WebApp1       #Nome de usuário da conta GMSA
+      Scope: contoso.com  #Nome de domínio DNS
+  CmsPlugins:
+  - ActiveDirectory
+  DomainJoinConfig:
+    DnsName: contoso.com        #Nome de domínio DNS
+    DnsTreeName: contoso.com    #Nome de domínio DNS raiz
+    Guid: 244818ae-87ac-4fcd-92ec-e79e5252348a  #GUID
+    MachineAccountName: WebApp1 #Nome de usuário da conta GMSA
+    NetBiosName: CONTOSO        #Nome de domínio NETBIOS
+    Sid: S-1-5-21-2126449477-2524075714-3094792973 #SID da GMSA
+```
+
+O recurso de especificação de credencial acima deve ser salvo como 
+`gmsa-Webapp1-credspec.yaml` e aplicado no cluster usando: 
+`kubectl apply -f gmsa-Webapp1-credspec.yml`
+
+## Configure um ClusterRole para habilitar RBAC nas especificações de credenciais GMSA específicas
+
+Uma ClusterRole precisa ser definido para cada recurso de especificação 
+de credencial GMSA. Isto autoriza o verbo `use` em um recurso GMSA específico
+por um sujeito, geralmente uma conta de serviço. O exemplo a seguir mostra 
+um ClusterRole que autoriza o uso de credencial `gmsa-WebApp1` 
+acima. Salve o arquivo como gmsa-webapp1-role.yaml e aplique 
+usando `kubectl apply -f gmsa-webapp1-role.yaml`
+
+```yaml
+#Criando um Role para ler o credspec
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: webapp1-role
+rules:
+- apiGroups: ["windows.k8s.io"]
+  resources: ["gmsacredentialspecs"]
+  verbs: ["use"]
+  resourceNames: ["gmsa-WebApp1"]
+```
+
+## Atribua o Role às contas de serviço para usar especificações de credencial GMSA específicas
+
+Uma conta de serviço (com a qual os Pods virão configurados), precisa ser vinculada 
+ao ClusterRole criado acima. Isto autoriza a conta de serviço a usar a especificação apropriada 
+de recurso de credencial GMSA. O trecho a seguir mostra a conta de serviço padrão vinculada ao ClusterRole `webapp1-role`, para usar a especificação 
+de recurso de credencial `gmsa-WebApp1` criada acima.
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: allow-default-svc-account-read-on-gmsa-WebApp1
+  namespace: default
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: default
+roleRef:
+  kind: ClusterRole
+  name: webapp1-role
+  apiGroup: rbac.authorization.k8s.io
+```
+
+## Configure a especificação de recurso de credencial GMSA em uma especificação de Pod
+
+O campo de especificação do Pod `securityContext.windowsOptions.gmsaCredentialSpecName`,  
+é usado para especificar referências para recursos customizados, em especificações 
+de certificado GMSA apropriadas em especificações do Pod. 
+Isto configura todos contêineres na especificação do Pod para usar a especificação GMSA. 
+Uma amostra da especificação do Pod com anotação populada para referir-se a `gmsa-WebApp1`:
```

Comment:
@stormqueen1990 and @DonatoHorn  I try remove "especificar/cões/ção".

"O campo `securityContext.windowsOptions.gmsaCredentialSpecName` do Pod,  
é usado de referência para recursos customizados, em especificações 
de certificado GMSA apropriadas em especificações do Pod. 
Isto configura todos contêineres do Pod para usar GMSA. 
Uma amostra com anotação populada para referir-se a `gmsa-WebApp1`:"